### ISSUE TITLE
The live demo page is 404

### ISSUE THREAD

**Issue by aviaryan** [heartz/india-intl-crypto-rates#1]
https://moonlit-academy-178514.appspot.com/

**Comment by heartz** [comment]
Yeah I took it down a few weeks ago, have not been working on this actively.

**Comment by aviaryan** [comment]
I see. Is the project in working condition now i.e. does every feature work? I would love to contribute to it and have it back online so that people can use it. 🙂

**Comment by heartz** [comment]
It should be, haven't pushed any broken code. There is alot of code duplication which I had intended to remove but never got back to it. Feel free to fork and host it.
